packages/analytics/analytics-metric-provider/vite.config.ts

```diff
@@ -18,12 +18,19 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
     },
     rollupOptions: {
       // Make sure to externalize deps that shouldn't be bundled into your library
-      external: ['@kong-ui-public/i18n', '@kong-ui-public/analytics-utilities'],
+      external: [
+        '@kong-ui-public/i18n',
+        '@kong-ui-public/analytics-config-store',
+        '@kong-ui-public/analytics-utilities',
+        'pinia',
+      ],
       output: {
         // Provide global variables to use in the UMD build for externalized deps
         globals: {
           '@kong-ui-public/i18n': 'kong-ui-public-i18n',
+          '@kong-ui-public/analytics-config-store': 'kong-ui-public-analytics-config-store',
           '@kong-ui-public/analytics-utilities': 'kong-ui-public-analytics-utilities',
+          pinia: 'pinia',
```

Comment:
I believe vue should still be in external and globals

Reply:
Same comment as above; it's already externalized in the shared config.

Reply:
forgot they're deep merged 😅